### PR TITLE
Bugfix/my habit new style 320px

### PR DESCRIPTION
**Pull request by nvolodymyr**

### CODE REVIEW

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.scss [hunk 1]
```diff
@@ -106,3 +110,58 @@ $second-color-font: #839C94;
 .star {
   background: url("../../../../../../../assets/img/icon/star-circle.png") no-repeat center;
```
Comment (mplesha) [inline src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.scss:111]: Improving long relative paths import:
here and below absolute path to "assets/img/" looks better then "../../.."

**Comment by mplesha** [comment]
improve it:
```css
 .calendar-img {
      width: 20px;
      height: 20px;
      background: $bg-color url("../../../../../../../assets/img/icon/calendar.png") no-repeat center;
    }
```

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.scss (hunk 1 above)
Comment (nvolodymyr) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.scss]: done

**Comment by nvolodymyr** [comment]
> improve it:
> 
> ```css
>  .calendar-img {
>       width: 20px;
>       height: 20px;
>       background: $bg-color url("../../../../../../../assets/img/icon/calendar.png") no-repeat center;
>     }
> ```

**Pull request merged**